exporter/sinks/prom.go

```diff
@@ -50,25 +51,29 @@ func (s *PromRemoteWriteSink) Push(ctx context.Context, batch *pb.PodNetworkMetr
 		if dstIP.IsPrivate() {
 			dstIPType = "private"
 		}
+		labels := []promwrite.Label{
+			{Name: "__name__", Value: "egressd_transmit_bytes_total"},
+			{Name: "src_pod", Value: ensureNonEmpty(m.SrcPod)},
+			{Name: "src_node", Value: ensureNonEmpty(m.SrcNode)},
+			{Name: "src_namespace", Value: ensureNonEmpty(m.SrcNamespace)},
+			{Name: "src_zone", Value: ensureNonEmpty(m.SrcZone)},
+			{Name: "src_ip", Value: ensureNonEmpty(m.SrcIp)},
+
+			{Name: "dst_pod", Value: ensureNonEmpty(m.DstPod)},
+			{Name: "dst_node", Value: ensureNonEmpty(m.DstNode)},
+			{Name: "dst_namespace", Value: ensureNonEmpty(m.DstNamespace)},
+			{Name: "dst_zone", Value: ensureNonEmpty(m.DstZone)},
+			{Name: "dst_ip", Value: dstIP.String()},
+			{Name: "dst_ip_type", Value: dstIPType},
+			{Name: "cross_zone", Value: isCrossZoneValue(m)},
+
+			{Name: "proto", Value: protoString(uint8(m.Proto))},
+		}
+		sort.Slice(labels, func(i, j int) bool {
+			return labels[i].Name < labels[j].Name
+		})
```

Comment:
@123BLiN From pure performance point of view consider using pre-sorted labels here (as [you already did in a test](https://github.com/castai/egressd/pull/109/files#diff-4ee7f1b77c14090b7b7b96fd5574a1e59fe437410fd958a8c8ec2bc31ad20708R63-R76))
And in prom_test you can take this Slice, sort it, and verify that it did not change after sorting.